artifacts/rhdh-plugins-reference/quickstart/quickstart-plugin-user.adoc

```diff
@@ -0,0 +1,72 @@
+[id="using-quickstart-plugin"]
+= Using the Quickstart plugin
+
+The Quickstart plugin provides a guided onboarding experience for new users of {product}. It displays a customizable drawer interface with interactive quickstart steps that help users get familiar with the platform.
+
+== Features
+Interactive Drawer Interface:: Displays quickstart steps in a slide-out drawer.
+Progress Tracking:: Tracks completion status of individual steps with persistent storage.
+Configurable Content:: Define custom quickstart items through app configuration.
+Visual Progress Indicator:: Shows overall completion progress with a progress bar.
+Call-to-Action Support:: Each step can include clickable action buttons.
+
+.Procedure
+. Open the Quickstart drawer
+.. Navigate to the **top right corner** of the {product-short} interface.
+.. Click the icon labeled **“Quick start”** (represented by a question mark icon in the top menu).
+.. The drawer panel on the **right-hand side** of the screen slides open.
+
+. Review the available Quickstart items
++
+In the drawer, you will see a list of onboarding tasks, including:
+
+Set up authentication::
+Configure secure login credentials to protect your account.  
+
+Configure RBAC::  
+  Set up Role-Based Access Control for your team.
+
+Configure Git::  
+  Connect your Git repositories for source control and automation.
+
+Manage plugins::  
+  Enable or disable additional functionality in the Hub.
++
+Each item may be expandable and interactive.
+
+. Track Your Progress
+* Each Quick Start item includes a **status indicator**, such as:
+  ** _Not started_
+  ** _In progress_
+  ** _Completed_
++
+As you complete each step or task, your progress is automatically tracked.
+
+[NOTE]
+====
+Quickstart items are **configurable**, meaning the content and order can vary depending on your Red Hat Developer Hub instance.
+
+You can **revisit** or **reset** tasks at any time for review or team onboarding.
```

Comment:
What does reset mean here? Resetting the steps to not started or updating the steps in app-config?